src/router/index.js

```diff
@@ -1,14 +1,19 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
-import Home from '../views/Home.vue'
+// import Home from '../views/Home'
 
 Vue.use(VueRouter)
 
 const routes = [
+  // {
+  //   path: '/',
+  //   name: 'Home',
+  //   component: Home
+  // },
```

Comment:
Took Home out of the routes because it was putting a new instance of home inside the student feedback section. We still have access to it, though (like by clicking on the logo).